examples/plotting.py

```diff
@@ -0,0 +1,84 @@
+# Note: some of the code in the present code block is derived from another project licensed under The MIT License (MIT),
```

Comment:
1. Please add the full Apache 2.0 header license comment. 
2. Add a title and a description in the `"""`. THIS IS IMPORTANT because otherwise the documentation website won't compile as in #147. See [here](https://github.com/Neuraxio/Neuraxle/commit/c3e11aaaefef5c10bb86aedd3cf9f5316d39aef7) an example of how you can fix this. 
3. Move the plotting functions to a Neuraxle package to import them. This way, the notebook will be able to import them. Add this MIT comment below the Apache 2.0 comment, but still in the `"""` section and as a `.. ` RST docstring comment just like the License. This way, the MIT comment will only be in the file under `./neuraxle/...py` file and won't have to be in the `./example/....py` file.